.travis.yml

```diff
@@ -21,6 +21,6 @@ notifications:
 before_install:
   - pip install --user codecov
 after_success:
-  - mvn validate -e
+  - travis_wait mvn validate -e
```

Comment:
I've been getting build failures because some packages take a while to install.
https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received